README.md

```diff
@@ -31,9 +31,7 @@ import (
 )
 
 func main() {
-    oauthToken := "xxxxxxx"
-    ts := oauth2.StaticTokenSource(&oauth2.Token{AccessToken: oauthToken})
-    tc := oauth2.NewClient(context.Background(), ts)
+    tc := dnsimple.StaticTokenClient(context.Background(), "your-token")
```

Comment:
Given this is an HTTP client, consider including that in the name of the function. `StaticTokenHTTPClient`